src/clj_kondo/impl/linters.clj

```diff
@@ -475,6 +475,28 @@
            (node->line filename node
                        finding-type msg)))))))
 
+(defn lint-used-underscored-bindings! [ctx]
```

Comment:
I actually think we can keep what you did before, linting the used and unused bindings in one go. This is better for performance probably, so let's revert that. We might just rename the function to `lint-bindings`.